eslint.yaml.config.mjs

```diff
@@ -0,0 +1,35 @@
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { FlatCompat } from '@eslint/eslintrc'
+import js from '@eslint/js'
+import yamlEslintParser from 'yaml-eslint-parser'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+export default [
+  ...compat.extends(
```

Comment:
I dont think the compat is needed here. https://github.com/ota-meshi/eslint-plugin-yml?tab=readme-ov-file#new-config-eslintconfigjs

Reply:
Yup I forgot to check it updated now